src/Perspex.SceneGraph/Media/Pen.cs

```diff
@@ -42,13 +58,20 @@ public Pen(uint color, double thickness, IReadOnlyList<double> dashArray = null)
         public Brush Brush { get; }
 
         /// <summary>
-        /// Gets the length of alternating dashes and gaps.
-        /// </summary>
-        public IReadOnlyList<double> DashArray { get; }
-
-        /// <summary>
         /// Gets the stroke thickness.
         /// </summary>
-        public double Thickness { get; }
+        public double Thickness { get; } = 1.0;
+
+        public DashStyle DashStyle { get; }
```

Comment:
These members could do with documentation, as the rest of the class has it. However, I can do that if you don't want to.